lib/gateway/Shard.js

```diff
@@ -972,6 +972,28 @@ class Shard extends EventEmitter {
                 }
                 break;
             }
+            case "VOICE_CHANNEL_STATUS_UPDATE": {
+                let channel = this.client.getChannel(packet.d.id);
+                if(!channel) {
+                    break;
+                }
+                const oldChannel = {
+                    status: channel.status
+                };
+                channel = channel.update({
+                    ...channel,
+                    status: packet.d.status
+                });
```

Comment:
No need to spread channel update or even to reassign `channel` here, it'll check if the props exist anyways.